docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+  server:
+    build: /community_portal
+    command: bash -c "python community_portal/manage.py makemigrations && python community_portal/manage.py migrate && python community_portal/manage.py runserver 0.0.0.0:8000"
+    container_name: community_portal
+    volumes:
+      - .:/community_portal
+    ports:
+      - "8000:8000"
+  client:
+    build: /client
+    command: flutter run -d chrome
```

Comment:
There is no flutter command in the container. It contains only the built assets.

Reply:
Try something like that:
```
version: '3'

services:
  server:
    build: ./community_portal
    command: bash -c "python community_portal/manage.py makemigrations && python community_portal/manage.py migrate && python community_portal/manage.py runserver 0.0.0.0:8000"
    container_name: community_portal
    volumes:
      - .:/community_portal
    ports:
      - "8000:8000"
  client:
    build: ./client
    container_name: client
    volumes:
      - .:/client
    ports:
      - "8001:80"
    environment:
      - NGINX_PORT=80
```

Reply:
![image](https://user-images.githubusercontent.com/96618001/174637525-2f7e87df-664e-4cf9-a718-4e5b060ade73.png)
look!

Reply:
Thank you so much ! :)